ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/lldp.yml

```diff
@@ -20,19 +20,19 @@ ethernet_interfaces:
       receive: false
 
   - name: Ethernet2
-    type: switched
     description: Switched port with no LLDP rx/tx
-    mode: access
-    vlans: 110
+    switchport:
+      mode: access
+      access_vlan: 110
     lldp:
       transmit: false
       receive: true
 
   - name: Ethernet3
-    type: switched
     description: No special LLDP settings
-    mode: access
-    vlans: 110
+    switchport:
```

Comment:
I would have expected `enabled: true` here so we also render the `switchport` on output. Should we make `enabled` key required, to force the user to decide?

Reply:
we have not implemented this in config template because it is a default setting, currently it generated config for `switchport.enabled: false` i.e `no switchport`

Reply:
Fixed